deployment/local-jvm/example-runner/runLocalJVMExample.sh

```diff
@@ -17,7 +17,7 @@
 FILE=example-runner/target/example-runner-*-exec.jar
 # Run the rest example
 if [ -f $FILE ]; then
-  java -Dspring.profiles.active=eureka,rest -jar $FILE
+  java -Dspring.profiles.active=rest,static,debug -jar $FILE
```

Comment:
I'm reading the list rest, static, debug as a list of profiles.  With that line of thought, I would expect a yaml file for each profile.  I don't see one for debug.